src/fio/fioEngine.js

```diff
@@ -824,6 +827,14 @@ export class FioEngine extends CurrencyEngine {
       if (!res) {
         throw new Error('Service is unavailable')
       }
+    } else if (actionName === 'getFioNames') {
+      res = await promiseNy(
+        this.currencyInfo.defaultSettings.apiUrls.map(apiUrl =>
+          timeout(this.fioApiRequest(apiUrl, actionName, params), 10000)
+        ),
+        fioGetNameChecker,
```

Comment:
Could `fioGetNameChecker` be inline/anonymous function instead? Might be worth while to keep things tidy by not having the extra import. Just suggestion, I'm indifferent.